platform/cri-o/deps.bzl

```diff
@@ -13,3 +14,14 @@ def cri_o_dependencies():
         ],
         patch_args = ["-p1"],
     )
+
+    go_repository_alt(
+        name = "com_github_containers_conmon",
+        commit = "1bddbf7051a973f4a4fecf06faa0c48e82f1e9e1", # 2.0.15
+        importpath = "github.com/containers/conmon",
+        build_file_generation = "off",
```

Comment:
If we're turning off build file generation, what's the point of having this be a go_repository rule instead of a git_repository rule?

Reply:
The git_repository rule seems to cause the patched-in symlink to no longer be a symlink for some reason, and I can't figure out why. In any case, I'll change the go_repository_alt back into a go_repository.

Reply:
Okay; we might want to do some digging on that at some point. If you don't want to do it now, can you file a low-priority issue?